python/lsst/ts/Dome/llc_configuration_limits/amcs_limits.py

```diff
@@ -1,5 +1,9 @@
+import math
+
 from .common_amcs_and_lwscs_limits import CommonAmcsAndLwscsLimits
 
+_DEGREES_TO_RADIANS = math.pi / 180.0
```

Comment:
If you do go with a constant instead of the functions @pkubanek suggested then I suggest you only define this in one module, as a public constant, and import it everywhere.

Reply:
I removed the constants everywhere and replaced them with the math functions, except in one case where using the numpy function produces clearer code.